### PR TITLE
Implement the sendText/1 predicate from the BWAPI

### PR DESCRIPTION
**Pull request by ThijsRay**
As mentioned [here](https://mattermost.ewi.tudelft.nl/p-mas-2017-2018/pl/fd4wak7k93rppf1gtawp8eye7c), people were in need of a send chat message connector.

The environment makes sure that:
- There is a maximum character limit of 140.
- There is a rate limit of 1 message per second.
- Cheat codes are not accepted.

### CODE REVIEW

**Comment by Venorcis** [comment]
Sorry, this will not be added because of the cheating potential and that I don't want to see chatting bots in my stream anyway ;)

File: eisbw/src/main/java/eisbw/actions/SendText.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+			"the gathering",
+			"noglues",
+			"power overwhelming",
+			"food for thought",
+			"there is no cow level",
+			"ophelia",
+			"staying alive",
+			"man over game",
+			"game over man",
+			"war aint waht it used to be",
```
Comment (TTvanWillegen) [inline eisbw/src/main/java/eisbw/actions/SendText.java]: Did you make a typo here or is this the actual cheat code?

File: eisbw/src/main/java/eisbw/actions/SendText.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+				|| text.toLowerCase().startsWith("protoss")) return false;
+
+		// Deny any slash commands.
+		if (text.toLowerCase().startsWith("/")) return false;
+
+		// Deny any cheats.
+		if (CHEATS.contains(text.toLowerCase())) return false;
+
+		// Only allow up to 1 message a second.
+		if (!this.rateLimit.tryAcquire()) return false;
```
Comment (TTvanWillegen) [inline eisbw/src/main/java/eisbw/actions/SendText.java]: This should probably be below text.length.

File: eisbw/src/main/java/eisbw/actions/SendText.java (hunk 1 above)
Comment (ThijsRay) [reply eisbw/src/main/java/eisbw/actions/SendText.java]: This seems to be a typo.

**Pull request closed without merging**